keyboards/electronlab/klor/config.h

```diff
@@ -0,0 +1,148 @@
+/*
+Copyright 2024 ElectronLab
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+// ┌─────────────────────────────────────────────────┐
+// │ p i n   c o n f i g                             │
+// └─────────────────────────────────────────────────┘ 
+
+#define I2C1_SCL_PIN GP3
+#define I2C1_SDA_PIN GP2
+
+// ┌─────────────────────────────────────────────────┐
+// │ k e y   m a t r i x                             │
+// └─────────────────────────────────────────────────┘ 
+
+#define MATRIX_ROWS 8
+#define MATRIX_COLS 6
+#define DEBOUNCE 5
+
+
+// ┌─────────────────────────────────────────────────┐
+// │ s p l i t   c o m m u n i c a t i o n           │
+// └─────────────────────────────────────────────────┘ 
+
+#define USE_SERIAL
+#define SERIAL_USE_MULTI_TRANSACTION
+#define SPLIT_USB_DETECT
+
+//Half Duplex communication
+#define SERIAL_USART_TX_PIN GP1     // USART TX pin
+
```

Comment:
```suggestion
```
Setting `split.serial.pin` in `keyboard.json` should be enough for the half duplex mode.

Reply:
Agreed. Change is done and committed.